### PR TITLE
revert resource increases

### PR DESCRIPTION
**Pull request by sdowell**
Reason for revert: The problems solved by these commits have been fixed
by using autopilot-specific defaults

### CODE REVIEW

**Review by karlkfi — APPROVED** [review]
/lgtm

Approved by @janetkuo in chat

File: pkg/reconcilermanager/controllers/reconciler_container_resources.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func ReconcilerContainerResourceDefaults() map[string]v1beta1.ContainerResources
 		reconcilermanager.HelmSync: {
 			ContainerName: reconcilermanager.HelmSync,
 			CPURequest:    resource.MustParse("50m"),
-			MemoryRequest: resource.MustParse("256Mi"),
```
Comment (janetkuo) [inline pkg/reconcilermanager/controllers/reconciler_container_resources.go:49]: @nan-yu I remember we bumped helm memory several times for Autopilot. Should we lower the number here?

Comment (nan-yu) [reply pkg/reconcilermanager/controllers/reconciler_container_resources.go:49]: We bumped helm cpu request for Autopilot, not the memory: https://github.com/GoogleContainerTools/kpt-config-sync/pull/378.
We could probably lower it from `50m` to `10m`.

**Pull request merged**